Shelfie.Identity/Shelfie.Identity.BusinessLogic/Enums/ErrorCode.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Shelfie.Identity.BusinessLogic.Enums;
+
+public enum ErrorCode
+{
+    EntityAlreadyExists,
```

Comment:
Why not separate error code per use case? Then there will be no generic file to maintain